templates/login_or_signup.tmpl

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+
+<html lang="en">
+    <head>
+        <meta charset="utf-8" />
+        <meta
+            name="description"
+            content="The best way to book club" />
+        <meta
```

Comment:
just curious what does this do?

Reply:
Yeah so I had to look this boilerplate shit up. I can talk about every line honestly, if you're curious about the other meta tags as well. This one sets the text that shows up under our site when you search for it on google or whatever. You can also see the description in the fb link popup
![screen shot 2018-03-08 at 1 17 44 pm](https://user-images.githubusercontent.com/549631/37168508-23adbb20-22d3-11e8-9a69-7bc47062fded.png)